app/script/view_model/MessageListViewModel.coffee

```diff
@@ -383,7 +383,7 @@ class z.ViewModel.MessageListViewModel
 
     @_track_context_menu message_et
 
-    if message_et.has_asset() and not message_et.is_expired()
+    if message_et.has_asset() and message_et.ephemeral_status() is z.message.EphemeralStatusType.NONE
```

Comment:
Now the message will be shown in other cases. I think `if message_et.has_asset() and not message_et.is_ephemeral()` is what we are aiming for.